web/eslint.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "modules": true
+        }
+    },
+    "extends": "eslint:recommended",
```

Comment:
The version im using is 18 if  bi is using this or a later version the rules may need testing.